server/src/main/java/org/candlepin/hostedtest/HostedTestSubscriptionServiceAdapter.java

```diff
@@ -11,7 +11,7 @@
  * Red Hat trademarks are not licensed under GPLv2. No permission is
  * granted to use or replicate Red Hat trademarks that are incorporated
  * in this software or its documentation.
- */
+ */ConsumerTypeRe
```

Comment:
This must have been pasted by mistake and should be removed.

Reply:
Fixed.